mllib/src/main/scala/org/apache/spark/ml/r/GeneralizedLinearRegressionWrapper.scala

```diff
@@ -200,3 +237,27 @@ private[r] object GeneralizedLinearRegressionWrapper
     }
   }
 }
+
+/**
+ * This utility transformer converts the predicted value of GeneralizedLinearRegressionModel
```

Comment:
This is an inherent feature of other classification algorithms who extends ```Classifier``` currently, only GLM use the separate conversion since ```GeneralizedLinearRegression extends Regressor```. So it should not be reused, and we can move it to RWrapperUtils.scala when it's necessary. Thanks.